src/commands/synthetics/__tests__/run-tests-lib.test.ts

```diff
@@ -105,7 +105,7 @@ describe('run-test', () => {
             ...partialCIConfig,
             publicIds: ['public-id-1', 'public-id-2'],
           })
-        ).rejects.toMatchError(new CiError('NO_TESTS_TO_RUN'))
+        ).rejects.toThrowError(new CiError('NO_TESTS_TO_RUN'))
```

Comment:
`toMatchError()` was a custom jest matcher, provided by `jest-matcher-specific-error`. This custom matcher is not required anymore.